hdx_hapi/db/dao/humanitarian_needs_view_dao.py

```diff
@@ -52,9 +53,10 @@ async def humanitarian_needs_view_list(
     if population_status:
         query = query.where(HumanitarianNeedsView.population_status == population_status)
 
-    if population:
-        query = query.where(HumanitarianNeedsView.population == population)
-
+    if population_min:
+        query = query.where(HumanitarianNeedsView.population >= population_min)
+    if population_max:
+        query = query.where(HumanitarianNeedsView.population <= population_max)
```

Comment:
it looks good, but I noticed that for other filters we are using < ..max instead of <=max

Reply:
OK - I changed this to < in the three places it occurs.